modules/op-changing-source-branch-in-repository-crd.adoc

```diff
@@ -0,0 +1,29 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="changing-source-branch-in-repository-crd_{context}"]
+= Changing the source branch for the pipeline definition
+
+[role="_abstract"]
+By default, when processing a push event or a pull request event, {pac} fetches the pipeline definition from the branch that triggered the event. You can use the `pipelinerun_provenance` setting in the `Repository` custom resource definition (CRD) to fetch the definition from the default branch configured on the Git repository provider, such as `main`, `master`, or `trunk`.
```

Comment:
```suggestion
By default, when processing a push event or a pull request event, {pac} fetches the pipeline definition from the branch that triggered the event. You can use the `pipelinerun_provenance` setting in the `Repository` custom resource (CR) to fetch the definition from the default branch configured on the Git repository provider, such as `main`, `master`, or `trunk`.
```

The Repository resource is referred to in the rest of the doc as a CR, not a CRD. They are two different things:
https://docs.openshift.com/aro/3/dev_guide/creating_crd_objects.html#crd_dev-guide-crd
Pretty sure CR is correct here.

Reply:
Unfortunately the CRD term is used in the existing doc, who approved this PR too, and I can't cycle back to developers because of time constraints, so again this goes into the restructure that is already planned.